src/components/PortfolioList.tsx

```diff
@@ -17,7 +18,7 @@ const PortfolioList = () => {
     const [table, setTable] = useState([])
 
     const handleTable = () => {
-        axios.get('http://3.236.213.150:8081/portfolios/users/all/' + cookies['user'].id)
+        axios.get(url + '/portfolios/users/all/' + cookies['user'].id)
```

Comment:
@ddinh please store `url + "/portfolios"` to a variable and pass here to minimize code duplication throughout this file